src/Input/ToggleSwitch/Styles.ts

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components';
+import { ToggleBallProps, ToggleProps } from './ToggleSwitch.interface';
+
+const ToggleSwitchWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  position: relative;
+`;
+
+const Toggle = styled.div<ToggleProps>`
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  overflow: hidden;
+  position: relative;
+  transform: translate3d(0, 0, 0);
+  background-color: ${({
+    activeBackgroundColor,
+    inactiveBackgroundColor,
+    active,
+  }) => (active ? activeBackgroundColor : inactiveBackgroundColor)};
+  height: 25px;
+  width: 56px;
+  border-radius: 14px;
+`;
+
+const ToggleBall = styled.div<ToggleBallProps>`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  z-index: 2;
+  border-radius: 50%;
+  transition: transform 0.3s cubic-bezier(1, 0.19, 0.15, 0.7);
+  transition-delay: 0.1s;
+  will-change: transform;
```

Comment:
Hmm I didn't know about this but found this out on the docs https://developer.mozilla.org/en-US/docs/Web/CSS/will-change

```
Important: will-change is intended to be used as a last resort, in order to try to deal with existing performance problems. It should not be used to anticipate performance problems.
```

I don't foresee any performance issues so I suggest removing it.